app/src/main/res/values-it/strings.xml

```diff
@@ -234,4 +239,70 @@
+    <string name="setup_gesture_configuration_listening_found">Gesto riconosciuto. Configura ora le impostazioni a tuo piacimento, poi premi su successivo.</string>
+    <string name="setup_gesture_configuration_subtitle">Impostazioni</string>
+    <string name="setup_gesture_configuration_adjustment_sensitivity_desc">Una sensibilità minore vuol dire che sarà più difficile eseguire gesti sulla maggior parte dei dispositivi, mentre una sensibilità maggiore è più prona a false rilevazioni.</string>
+    <string name="setup_gesture_configuration_adjustment_model_desc">Aggiusta la dimensione del dispositivo da usare come modello per i gesti. Attualmente impostato: %1s</string>
+    <string name="configuration_button_troubleshooting">Aiuto</string>
+    <string name="configuration_button_next">Prossimo</string>
+    <string name="configuration_button_finish">Finito</string>
+    <string name="bs_troubleshooting_title">Aiuto</string>
+    <string name="bs_troubleshooting_content">Se i gesti non sono funzionanti, ecco alcune cose che puoi provare per farku fybzuibare:\n\n• Prova a cambiare la dimensione del dispositivo. Tap, Tap prova ad indovinare la dimensione giusta del dispositivo quando si sceglie il modello, ma non è sempre corretta\n\n• Cambia la sensibilità - alcuni dispositivi potrebbero aver bisogno di una sensibilità maggiore\n\n• Prova a fare il gesto in una posizione differente. Alcuni sensori sono locati in parti differenti del dispositivo, quindi potresti dover premere più in alto, più in basso o agli angoli.\n\n• Premi più forte. Tieni il telefono saldamente fermo mentre provi, per non farlo cadere. Una pressione più forte è più facile da riconoscere che una meno intensa.\n\n• Fai il doppio tocco velocemente, ma non troppo - se sei troppo lento potrebbe non essere registrato come doppio tap (il tempo di sensibilità è di circa un secondo)</string>
+    <string name="setup_foss_info_title">Tap, Tap è gratuito e Open Source</string>
+    <string name="setup_foss_info_content"><![CDATA[Tap, Tap è, e sarà sempre gratuito e costruito con componenti a sorgente aperta. Il codice sorgente è disponibile per chiunque voglia vederlo e apportare modifiche sulla <a href=\"https://kieronquinn.co.uk/redirect/TapTap/github\">repository GitHub</a>. I download ufficiali dell\'app è disponibile <b>solo</b> su <a href="https://kieronquinn.co.uk/redirect/TapTap/xda">XDA</a>, e sulla <a href="https://kieronquinn.co.uk/redirect/TapTap/github">repository GitHub</a>, e <a href="https://kieronquinn.co.uk/redirect/TapTap/apkmirror">APKMirror</a>. L\'app chiederà automaticamente di scaricare gli aggiornamenti da GitHub, ma il link potrebbe indirizzarti direttamente al download e potresti non vedere la pagina prima che il download cominci]]></string>
```

Comment:
- Prefer 'open source' to 'a sorgente aperta'
- Modify  'I download ufficiali dell\'app è disponibile' in 'I download ufficiali dell\'app sono disponibili'